appveyor.yml

```diff
@@ -102,7 +102,7 @@ test_script:
         $TestsFailed = $FALSE
         #netFx tests
         If ($env:TESTS -eq 'net') {
-            @('EnyimMemcached', 'Prevalence', 'RtMemoryCache', 'SysCache', 'SysCache2', 'CoreMemoryCache', 'CoreDistributedCache') | ForEach-Object {
+            @('EnyimMemcached', 'Prevalence', 'RtMemoryCache', 'SysCache', 'SysCache2', 'CoreMemoryCache', 'CoreDistributedCache', 'Common', 'Util.JsonSerializer') | ForEach-Object {
```

Comment:
The current logic does not support the two added projects case. It searches them in
```
C:\projects\nhibernate-caches\Common\NHibernate.Caches.Common.Tests\bin\Debug\net461\NHibernate.Caches.Common.Tests.dll
C:\projects\nhibernate-caches\Util.JsonSerializer\NHibernate.Caches.Util.JsonSerializer.Tests\bin\Debug\net461\NHibernate.Caches.Util.JsonSerializer.Tests.dll
```
Instead of:
```
C:\projects\nhibernate-caches\NHibernate.Caches.Common.Tests\bin\Debug\net461\NHibernate.Caches.Common.Tests.dll
C:\projects\nhibernate-caches\Util\NHibernate.Caches.Util.JsonSerializer.Tests\bin\Debug\net461\NHibernate.Caches.Util.JsonSerializer.Tests.dll
```

The easiest for them is to add additional direct `nunit3-console` calls.

Reply:
For completeness, Nant build files have to be completed for also running these tests when running `nant test` from the root.